apps/fis/fis-cos-api/fis-cos-api.yaml

```diff
@@ -8,10 +8,6 @@ spec:
   values:
     java:
       image: hmctspublic.azurecr.io/fis/cos-api:prod-c9e5ffd-20230314190321 #{"$imagepolicy": "flux-system:fis-cos-api"}
-      memoryRequests: "2048Mi"
-      cpuRequests: "1000m"
-      memoryLimits: "4096Mi"
-      cpuLimits: "2000m"
```

Comment:
sorry if I confused you but is there a reason you're removing these from the base file?
I only meant for you to remove from per environment overrides.

(unless these are too much which they do look quite high)

Reply:
oh ok. sorry i might have misunderstood you.. yes. we would need them in base file with that figure. as we are allowing upto 10 files to be uploaded before case is submitted. And the UI will be accessible by Citizens/Legal Professionals/Police

Reply:
that seems like a lot of resources, are you keeping the files in memory the whole time rather than storing them somewhere?